components/local/roc/src/roc.py

```diff
@@ -56,9 +63,9 @@ def main(argv=None):
   if args.target_lambda:
     df['target'] = df.apply(eval(args.target_lambda), axis=1)
   else:
-    df['target'] = df['target'].apply(lambda x: 1 if x == args.trueclass else 0)
-  fpr, tpr, thresholds = roc_curve(df['target'], df[args.trueclass])
-  roc_auc = roc_auc_score(df['target'], df[args.trueclass])
+    df['target'] = df['target'].apply(lambda x: 1 if x == trueclass else 0)
+  fpr, tpr, thresholds = roc_curve(df['target'], df[trueclass])
```

Comment:
Thanks! Didn't know that before.

Now it seems I have to use lambda because I want to make it case sensitive for default value "True".

Reply:
I changed my mind. It is case sensitive now. But since I need to support target_lambda, it may not always be a label. For example, the target_lambda could be: lambda x: x['a'] > x['b'].